### DISCUSSION TITLE
Instruction view

### DISCUSSION THREAD

**Discussion by Methew5** [sceneview/sceneform-android#257] [Q&A]
How can I change the content of the instruction?
I tried next code in arFragment:
val container = view.findViewById<ViewGroup>(R.id.handMotionLayout)
instructionsController.setView(TYPE_PLANE_DISCOVERY, layoutInflater.inflate(R.layout.ar_instruction, container, true))

Where ar_instruction:
<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
    android:layout_width="match_parent"
    android:layout_height="match_parent">

    <com.google.ar.sceneform.ux.HandMotionView
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        android:foregroundGravity="center"
        android:scaleType="center"
        android:src="@drawable/instruction" />

</FrameLayout>

But in this case, the instruction does not appear...

**Comment by ThomasGorisse** [comment]
Please use discussions for this.

You can simply override https://github.com/SceneView/sceneform-android/blob/26d7a4709160b9f65439de93586f17fdc1e60cbc/ux/src/main/res/layout/sceneform_instructions_plane_discovery.xml on your res folder.
